scripts/mesoscale/plots/exevs_mesoscale_atmos_sref_cape_plots.sh

```diff
@@ -241,10 +240,14 @@ done
 
 tar -cvf evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar *.png
 
+
 if [ $SENDCOM = YES ] ; then
  cp  evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar  $COMOUT/$STEP/$COMPONENT/$RUN.$VDATE/.  
 fi
 
+if [ $SENDDBN = YES ] ; then
+     $DBNROOT/bin/dbn_alert MODEL EVS_RZDM $job $COMOUT/evs.plots.sref.cape.past${past_days}days.v${VDATE}.tar
```

Comment:
@BinbinZhou-NOAA thanks for moving the SENDDBN section below the SENDCOM section. Please look at the path for the tar file, though. I believe it needs to match the path for the SENDCOM section. I think the line needs to be:
```
$DBNROOT/bin/dbn_alert MODEL EVS_RZDM $job $COMOUT/$STEP/$COMPONENT/$RUN.$VDATEevs.plots.sref.cape.past${past_days}days.v${VDATE}.tar
```

Reply:
Yes, I have fixed this error.

Thanks!